modules/460-log-shipper/hooks/internal/vector/transform/transforms_test.go

```diff
@@ -27,21 +27,28 @@ import (
 
 	"github.com/deckhouse/deckhouse/modules/460-log-shipper/hooks/internal/impl"
 	"github.com/deckhouse/deckhouse/modules/460-log-shipper/hooks/internal/v1alpha1"
+	"github.com/deckhouse/deckhouse/modules/460-log-shipper/hooks/internal/vector/model"
 )
 
-func loadMock(t *testing.T, parts ...string) string {
-	content, err := os.ReadFile(filepath.Join(append([]string{"testdata"}, parts...)...))
+func compareMock(t *testing.T, data []byte, parts ...string) {
+	filename := filepath.Join(append([]string{"testdata"}, parts...)...)
+	content, err := os.ReadFile(filename)
 	require.NoError(t, err)
 
-	return string(content)
+	if os.Getenv("D8_LOG_SHIPPER_SAVE_TESTDATA") == "yes" {
```

Comment:
The `golden` flag for test is more comfortable than env variable. But it's up to you

Reply:
I assume there are already environment variables that control tests behavior. That is why I went this way.